recipes/symengine/bld.bat

```diff
@@ -0,0 +1,19 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
```

Comment:
> Using `cmake -G "%CMAKE_GENERATOR%"` worked earlier, now it should be changed to `cmake -G %CMAKE_GENERATOR%`.

Sorry @isuruf I don't have enough experience with `cmake` nor Windows to help you there.